Jenkinsfile.central

```diff
@@ -0,0 +1,73 @@
+#!groovy
+
+node{
+   properties(
+       [parameters(
+           [string(defaultValue: 'develop', description: 'The SHA for the commit you wish to deploy.', name: 'GIT_COMMIT', trim: false),
+            string(defaultValue: '', description: 'The tag of the current release.', name: 'RELEASE_TAG', trim: false)]
```

Comment:
@hectcastro I think that reusing the ref for `checkout` as validation alone doesn't work because you could still release any commit to Maven Central by mistake. 

If we pair reusing the ref for `checkout` with doing a regex on `params.RELEASE_TAG`, then that seems solid. 

One problem that comes to mind is that we deploy the current state of the `develop` branch in the [Raster Foundry Release Pipeline](
https://github.com/azavea/raster-foundry-deployment/wiki/Release-Workflow#deploy-the-current-state-of-the-develop-branch), which is a different state than using the `x.x.x` tag, right?

Maybe we should just do regex on `params.RELEASE_TAG` and still allow specifying a git commit, that way we can mirror the other release pipeline?

Reply:
> One problem that comes to mind is that we deploy the current state of the develop branch in the Raster Foundry Release Pipeline, which is a different state than using the x.x.x tag, right?

Yes. We should be using the commit that is tagged instead. The tagged commit will be a merge commit of the release's changes being merged into `master`.

I think that Chris' comment above is assuming that the platform release is completed (thus, a tag exists), then the ref of that tag is fed to this pipeline so that its artifacts go into MC.

Reply:
Any objection @notthatbreezy?

Reply:
This looks good to me!

Reply:
Just realized, we'll still need to retain a field for passing `GIT_COMMIT` in `Jenkinsfile` because we pull down container images built in `develop` at the last `SHA` and retag them for prod.

Reply:
Evaluating `Jenkinsfile.release`, it doesn't look like there's any need for the initial `checkout`, is there?

https://github.com/raster-foundry/raster-foundry/blob/4ba02c9a1051fa0c8de0b289bd9b3fd345cccbb7/Jenkinsfile.release#L26